app.rb

```diff
@@ -12,9 +12,9 @@
 require 'net/http'
 
 # Load Environment Variables
-Prius.load(:gocardless_token)
-Prius.load(:gc_api_key_secret)
+Prius.load(:gc_access_token)
 Prius.load(:gc_creditor_id)
+Prius.load(:gc_environment)
```

Comment:
I've added this so you're more explicit about which environment you're using. You'll need to add it to Heroku, too.